src/sidebar.js

```diff
@@ -57,7 +57,7 @@ class Sidebar extends React.Component {
 
     this.state = {
       // the detected width of the sidebar in pixels
-      sidebarWidth: 0,
+      sidebarWidth: props.sidebarWidth,
```

Comment:
Can you rename your prop to `defaultSidebarWidth` to be in line with the naming pattern of uncontrollable components.